src/main/webapp/resources/js/admin-abonnement.js

```diff
@@ -119,6 +139,17 @@ const abonnementsFunctions = function () {
         });
     });
 
+    $(document.body).on('click', '.btn-assign', function() {
+        let idAbonnement = getId(this);
+        let path = 'adm-abonnement/' + idAbonnement;
+        $.ajax({
+            url: path,
+            success: function(data) {
+                $(".abonnementId").val(data.id);
```

Comment:
You already have an id, u do not need new ajax request. Just $(".abonnementId").val(idAbonnement);